exec/java-exec/src/main/resources/rest/static/js/querySubmission.js

```diff
@@ -95,7 +95,7 @@ function submitQuery() {
         data: $("#queryForm").serializeArray(),
         success: function (response) {
             closePopup();
-            var newDoc = document.open("text/html", "replace");
+            var newDoc = document.getElementById('query-results').contentWindow.document;
```

Comment:
wondering how we can make this use the variables defined in the ftl file so it can be programatically changed?